tools/ci_build/github/azure-pipelines/templates/stages/mac-ios-packaging-build-stage.yml

```diff
@@ -19,6 +19,8 @@ stages:
 
     variables:
       xcodeVersion: "14.3.1"
+      simulatorRuntimeVersion: "16.4"
```

Comment:
Can we capture in a comment how and when this value should be updated?

Reply:
good idea, added comments